k-distribution/tutorial/1_k/1_lambda/lesson_9/lambda.md

```diff
@@ -1,4 +1,6 @@
-// Copyright (c) 2014-2019 K Team. All Rights Reserved.
+---
+copyright: Copyright (c) 2014-2020 K Team. All Rights Reserved.
+---
 
 // C/Java-like comments like this one are completely ignored by the K tool.
```

Comment:
let's remove this comment and replace it with something talking about how markdown works, because this is not accurate otherwise.